api/src/main/java/io/opencensus/trace/Span.java

```diff
@@ -158,6 +158,20 @@ public final void addAnnotation(String description) {
   public abstract void addLink(Link link);
 
   /**
+   * Sets the {@link Status} to the {@code Span}.
+   *
+   * <p>If used, this will override the default {@code Span} status. Default is {@link Status#OK}.
+   *
+   * <p>If called multiple times, or status set via
```

Comment:
Is this a comment about the EndSpanOptions? I think it is important to explain how the two status methods interact.

Reply:
Done.